components/brave_wallet_ui/page/screens/fund-wallet/deposit-funds.tsx

```diff
@@ -74,6 +74,7 @@ export const DepositFundsScreen = () => {
   const selectedAccount = useSelector(({ wallet }: { wallet: WalletState }) => wallet.selectedAccount)
   const fullTokenList = useSelector(({ wallet }: { wallet: WalletState }) => wallet.fullTokenList)
   const networkList = useSelector(({ wallet }: { wallet: WalletState }) => wallet.networkList)
+  const selectedAsset = useSelector(({ wallet }: { wallet: WalletState }) => wallet.selectedDepositAsset)
```

Comment:
Couldn't we use router params for this instead of adding more actions/state to redux?

Reply:
Moved to use a Router Param!